deepfence_ui/app/scripts/actions/app-actions.js

```diff
@@ -1371,6 +1372,15 @@ export function listRegistryImagesAction(params) {
   return genericThunkAction(actionTypes, listRegistryImages, params);
 }
 
+export function getAccountIdAction() {
```

Comment:
maybe rename to `getConsoleAWSAccountId`

Reply:
renamed